components/formCreator/ErrorMessage.tsx

```diff
@@ -0,0 +1,13 @@
+import { QuestionValidationError } from '@/types/forms';
```

Comment:
urgh

Reply:
this used to exist at one point. i have no idea where the type went